dotnet/src/SemanticKernel.Abstractions/AI/ChatClient/ChatMessageExtensions.cs

```diff
@@ -46,4 +46,15 @@ internal static ChatMessageContent ToChatMessageContent(this ChatMessage message
 
         return result;
     }
+
+    /// <summary>Converts a list of <see cref="ChatMessage"/> to a <see cref="ChatHistory"/>.</summary>
+    internal static ChatHistory ToChatHistory(this IEnumerable<ChatMessage> chatMessages)
```

Comment:
Should there be public bidirectional helpers somewhere, for both `IEnumerable<ChatMessage>` to `ChatHistory` and vice versa?

Reply:
I will keep it internal before investigate what would be a case for a `ChatHistory : IEnumerable<ChatMessage>`, if this works, no conversion will be needed.